test/checks/label/implicit.js

```diff
@@ -39,4 +41,17 @@ describe('implicit-label', function () {
 		assert.isFalse(checks['implicit-label'].evaluate(node, {}, virtualNode));
 	});
 
+	it('should return false if label is empty for select', function () {
+		var params = checkSetup('<label>' +
+	    '<span class="label"></span>' +
+	    '<select id="target">' +
+	        '<option value="1" selected="selected">Please choose a region</option>' +
+	        '<option value="2">Coastal</option>' +
+	        '<option value="3">Forest</option>' +
+	        '<option value="4">Grasslands</option>' +
+	        '<option value="5">Mountains</option>' +
+	    '</select>' +
+		'</label>');
+		assert.isFalse(checks['implicit-label'].evaluate.apply(checkContext, params));
+	})
 });
```

Comment:
I think you should add the other cases from my markup as unit tests. I think you will find that some of them will fail (see comment below)

Reply:
There are already tests that assert nested cases and references...but I'll look again.

Reply:
I added the two unique test cases–they still pass.